adafruit_pyportal.py

```diff
@@ -649,6 +654,47 @@ def image_converter_url(image_url, width, height, color_depth=16):
                                           width, height,
                                           color_depth, image_url)
 
+    def io_push(self, feed, data):
+        # pylint: disable=line-too-long
+        """Push data to an adafruit.io feed
+
+        :param str feed: Name of feed to push data to.
+        :param data: data to send to feed
+
+        """
+        # pylint: enable=line-too-long
+
+        try:
+            aio_username = secrets['aio_username']
+            aio_key = secrets['aio_key']
+        except KeyError:
+            raise KeyError("\n\n")
```

Comment:
Add a more verbose with the `KeyError`, something like "Adafruit IO secrets are kept in secrets.py, please add them there!" works